### PR TITLE
add: security protocol in multiaddr

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

- Addressing-based material for the performance optimization of the security protocol being included in a peer's multiaddr.
- resolves of #214, part of #212

## Latest preview

Please view the latest Fleek preview [here](https://bafybeiavueq6icj3ouf6dopemoamvmmiuvnf5tuextxe5fynf5eyufb3ni.on.fleek.co/).

### CODE REVIEW

File: content/concepts/fundamentals/addressing.md [hunk 1]
```diff
@@ -10,28 +10,69 @@ aliases:
...
+## Security protocols
 
-For more detail, see the [Addressing spec](https://github.com/libp2p/specs/blob/master/addressing/README.md).
+In libp2p, establishing a secure connection between two peers involves negotiating a security protocol.
+Historically, this negotiation occurred in-band, meaning that it was conducted as part of the connection establishment
+process using the [multistream-select protocol](https://github.com/multiformats/multistream-select).
+However, this approach is susceptible to man-in-the-middle attacks, as a malicious actor could modify the list of supported
+handshake protocols to force a downgrade to a less secure protocol.
+
+To address this issue, libp2p recommends that peers encode the security protocol they wish to use directly in the multiaddr
```
Comment (MarcoPolo) [inline content/concepts/fundamentals/addressing.md:58]: We don't do this today.

**Review by MarcoPolo — COMMENTED** [review]
We don't do this today. I don't think we should merge this until/unless we make these changes.

**Comment by DannyS03** [comment]
> We don't do this today. I don't think we should merge this until/unless we make these changes.

Right, this is why I put the blocked label.

**Comment by marten-seemann** [comment]
> > We don't do this today. I don't think we should merge this until/unless we make these changes.
> 
> Right, this is why I put the blocked label.

Can we close it instead? I suggest not to write docs pages for features that we don't have yet, unless that feature is under active development and the rollout is imminent (as it was, for example, with WebRTC and Early Muxer Negotiation).

**Comment by DannyS03** [comment]
> > > We don't do this today. I don't think we should merge this until/unless we make these changes.
> > 
> > 
> > Right, this is why I put the blocked label.
> 
> Can we close it instead? I suggest not to write docs pages for features that we don't have yet, unless that feature is under active development and the rollout is imminent (as it was, for example, with WebRTC and Early Muxer Negotiation).

I was under the impression that it was active. Sure, we can close this and reopen if necessary.

**Pull request closed without merging**